app/src/main/java/org/mozilla/fenix/crashes/CrashReporterController.kt

```diff
@@ -55,10 +53,7 @@ class CrashReporterController(
 
         components.useCases.tabsUseCases.removeTab(session)
         components.useCases.sessionUseCases.crashRecovery.invoke()
-        navController.nav(
```

Comment:
It looks like this extension fun was created to "safely" navigate. Are we sure we don't need this here anymore?